pkg/core/block/block.go

```diff
@@ -235,3 +237,18 @@ func (b *Block) UnmarshalJSON(data []byte) error {
 	_ = b.ConsensusData.Hash()
 	return nil
 }
+
+// ToStackItem converts Block to stackitem.Array
+func (b *Block) ToStackItem() stackitem.Item {
```

Comment:
Same thing here, let's not create a dependency on `stackitem` until we really need it.